modules/claims_api/app/controllers/concerns/claims_api/v2/disability_compensation_validation.rb

```diff
@@ -500,11 +500,21 @@ def validate_form_526_service_information!(target_veteran)
       end
 
       def validate_claim_date_to_active_duty_end_date!(service_information)
-        max_period = service_information['servicePeriods'].max_by { |sp| sp['activeDutyEndDate'] }
-        date = form_attributes['claimDate'] || Time.find_zone!('Central Time (US & Canada)').today
-        if Date.strptime(date.to_s, '%Y-%m-%d') > Date.strptime(max_period['activeDutyEndDate'], '%Y-%m-%d') + 180.days
+        ant_sep_date = form_attributes&.dig('serviceInformation', 'federalActivation', 'anticipatedSeparationDate')
+        unless service_information['servicePeriods'].nil?
+          max_period = service_information['servicePeriods'].max_by do |sp|
+            sp['activeDutyEndDate']
+          end
+        end
+        max_active_duty_end_date = max_period['activeDutyEndDate']
+        date = form_attributes['claimDate']
+        if ant_sep_date.present? && max_active_duty_end_date.present? &&
+           ((Date.strptime(max_period['activeDutyEndDate'], '%Y-%m-%d') +
+           180.days < Date.strptime(date.to_s, '%Y-%m-%d')) ||
+              (Date.strptime(ant_sep_date,
+                             '%Y-%m-%d') + 180.days < Date.strptime(date.to_s, '%Y-%m-%d')))
```

Comment:
@stiehlrod  If I submit a claim with `activeDutyEndDate` that is set to something like 2027-11-30  and an `anticipatedSeparationDate` set to a similar date, it submits correctly.  It could just be the wording on the ticket, but I wanted to make sure that is correct because reading the ticket I thought that those dates, both of them, would both need to be less then or equal to 180 days out, correct?  In case it helps this is what I sent in, let me know if I am off and I can resolve this.
```
...
      "serviceInformation": {
        "servicePeriods": [
          {
            "serviceBranch": "Public Health Service",
            "activeDutyBeginDate": "2008-11-14",
            "activeDutyEndDate": "2027-11-30",
            "serviceComponent": "Active",
            "separationLocationCode": "25077"
          }
        ],
        "confinements": [
          {
            "approximateBeginDate": "2016-06-11",
            "approximateEndDate": "2016-09-11"
          },
          {
            "approximateBeginDate": "2019-06",
            "approximateEndDate": "2019-07"
          }
        ],
        "federalActivation": {
          "anticipatedSeparationDate": "2026-10-03",
          "activationDate": "2023-02-10"
        },
...
```